MapboxDirections/MBRouteLeg.swift

```diff
@@ -40,6 +68,9 @@ open class RouteLeg: NSObject, NSSecureCoding {
         }
         source = decodedSource
         
+
+        annotation = nil
```

Comment:
The encode method encodes the attribute data, so the decode method should decode it.